src/components/StaticWebsite.ts

```diff
@@ -77,7 +79,26 @@ export class StaticWebsite extends CdkConstruct implements Construct {
             comment: `Identity that represents CloudFront for the ${id} static website.`,
         });
 
+        /**
+         * CloudFront function that manipulates the HTTP responses to add security headers.
+         */
+        const code = `function handler(event) {
```

Comment:
Could we import this content from a separate file? Means if someone is contributing they get a proper JS file to avoid things like syntax errors which are harder to spot when it's a string?

Reply:
Good point, but now that the code is dynamic that makes a bit less sense I think (I've updated the code).